scripts/testy_launch.py

```diff
@@ -117,140 +117,147 @@ def snapshot_exists(snapshot_id):
+# is launched successfully. In the successful case, the dictionary also contains information
+# about the newly launched instance. Upon failure, the 'status' field is non-zero and the 'msg'
+# field contains the error message.
+def launch_from_distro(distro):
 
-    user = None
+    if not launch_template_exists(distro):
+        return {"status": 1, "msg": f"The distro '{distro}' does not exist."}
+
+    print("", end=f"\rCreating a {distro} testy server in EC2 ... ", flush=True)
     hostname = None
+    user = None
 
     try:
-        if not launch_template_exists(distro):
-            raise Exit(f"The distro '{distro}' does not exist.")
-        
         # Launch an EC2 instance based on a template.
         result = local(f"aws ec2 run-instances \
             --launch-template LaunchTemplateName={distro} \
-            --tag-specifications 'ResourceType=instance, Tags=[{{Key=Application,Value=testy}}, \
-                {{Key=LaunchTemplateName,Value={distro}}}]' \
             --query Instances[*].InstanceId \
             --output text", hide=True, warn=True)
         if result.stderr:
+            print("Failed.", flush=True)
             raise Exit(result.stderr)
+
+        print("Success!", flush=True)
         instance_id = result.stdout.strip()
+        wait_on_status_check(instance_id)
 
-        set_instance_name(instance_id, f"testy-{distro}-{instance_id}")
+        # Add 'Name' tags for the new instance and volume.
+        volume_id = get_volume_id_from_instance(instance_id)
+        instance_name = f"testy-{distro}-{instance_id.replace('-','')}"
+        set_tag_for_resource(instance_id, "Name", instance_name)
+        set_tag_for_resource(volume_id, "Name", f"testy-{distro}-{volume_id.replace('-','')}")
 
         hostname = get_hostname_for_instance(instance_id)
-        user = get_value_from_launch_template(distro, 'User')
+        user = get_tag_value_from_resource(instance_id, "User")
 
-        wait_instance_running(instance_id)
-        wait_instance_status_check_ok(instance_id)
     except Exception as e:
+        print("Failed.", flush=True)
         return {"status": 1, "msg": str(e).strip()}
 
-    return {"status": 0, "user": user, "hostname": hostname}
+    return {"status": 0, "user": user, "hostname": hostname,
+        "instance_id": instance_id, "instance_name": instance_name}
 
-# Launch an AWS instance given a snapshot ID.
-# This function returns a dictionary with a 'status' field that is set to 0 only when the instance
-# has been launched successfully. In that case, the dictionary contains a 'user' and 'host' fields.
-# Upon failure, the 'status' field is not 0 and the 'msg' field contains the error message.
-def testy_launch_snapshot(snapshot_id):
+# Launch an AWS instance from a snapshot ID.
+# This function returns a dictionary with a 'status' field that is set to 0 when the instance
+# is launched successfully. In the successful case, the dictionary also contains information
```

Comment:
ditto

Reply:
I removed this comment in [175bc3c](https://github.com/wiredtiger/testy/pull/50/commits/175bc3c6927b8d3eacdad0b7dbfdd805afbaf36c) as there's no need to have the same long comment twice.